cabal-install/bootstrap.sh

```diff
@@ -184,17 +184,22 @@ DEEPSEQ_VER="1.4.2.0"; DEEPSEQ_VER_REGEXP="1\.[1-9]\."
 
 case "$GHC_VER" in
     7.4*|7.6*)
+        # GHC 7.4 or 7.6
         BYTESTRING_VER="0.8.2.1"
```

Comment:
Yep, looks good now. Let's wait for the CI tests to pass, then you can squash the commits.